### ISSUE TITLE
Pasting in code doesn't work

### ISSUE THREAD

**Issue by mobluse** [Bushmills/jsforth#3]
I paste in code `: STAR  42 EMIT ;` and press Enter, but then `STAR` is not defined. If I type in the same text it works. The same problem exists in Firefox and Chrome in Windows 10. I cannot test the system further if it's not possible to paste in code and use it.

**Comment by Bushmills** [comment]
that's right. Pasting by mouse doesn't trigger the keyboard event handler, which is present to deal with interactive user input. For handling mouse events, such as pasting into the form, either a mouse event handler would be needed, or pasted text taken from the form. I haven't been successful getting a mouse event handler working properly (though mouse buttons and double clicks are caught and responded to), nor did I implement reading from form, as this complicates interactive KEY and KEY?, as those will probably be expected to respond to mouse paste operations too. Therefore current state of mouse pasting is: not implemented. An alternative may be: edit the screens file, using a text editor (which may support mouse pasting), then load source from there by using the LOAD word.
